Routers/BookingRouter.cs

```diff
@@ -0,0 +1,118 @@
+﻿using AirportTicketBooking.Classes;
+using AirportTicketBooking.Repositories;
+using ClassType = TicketClassType.ClassType;
+
+namespace AirportTicketBooking.Routers
+{
+    internal class BookingRouter
+    {
+        private BookingRepositoty _bookingRepositoty;
+        private FlightRouter _flightRouter;
+
+        public BookingRouter()
+        {
+            _bookingRepositoty = new BookingRepositoty();
+        }
+
+        public List<Booking> GetAllData()
+        {
+            return _bookingRepositoty.Load();
+        }
+
+        public List<Booking> GetAll()
+        {
+            return GetAllData().Where(booking => booking.ValidationMessage == null).ToList();
+        }
+
+        public bool TryGetByBookingId(int bookingId, out Booking booking)
```

Comment:
Usually, we use TryGet without specifying the name of the resource to get in the method because we know that this method will try to get an instance of the booking entity since this method is included in the Booking service.